File: app/src/main/java/com/example/edward/nyansapo/Begin_Assessment.java
package com.example.edward.nyansapo;

import android.app.ActivityOptions;
import android.content.Intent;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

public class Begin_Assessment extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_begin__assessment);
    }

    public void goHome(View v){
        Intent myIntent = new Intent(getBaseContext(), MainActivity.class);
        startActivity(myIntent, ActivityOptions.makeSceneTransitionAnimation(this).toBundle());
    }

    public void startPreAssessment(View v){
        Intent preIntent = new Intent(getBaseContext(), PreAssessment.class);
        startActivity(preIntent,ActivityOptions.makeSceneTransitionAnimation(this).toBundle());
    }
}

File: app/src/main/java/com/example/edward/nyansapo/Begin_Assessment.kt
package com.example.edward.nyansapo

import android.app.ActivityOptions
import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.example.edward.nyansapo.MainActivity
import com.example.edward.nyansapo.PreAssessment

class Begin_Assessment : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_begin__assessment)
    }

    fun goHome(v: View?) {
        val myIntent = Intent(baseContext, MainActivity::class.java)
        startActivity(myIntent, ActivityOptions.makeSceneTransitionAnimation(this).toBundle())
    }

    fun startPreAssessment(v: View?) {
        val preIntent = Intent(baseContext, PreAssessment::class.java)
        startActivity(preIntent, ActivityOptions.makeSceneTransitionAnimation(this).toBundle())
    }
}